core/core-domain/src/main/java/com/pgms/coredomain/domain/member/repository/MemberRepository.java

```diff
@@ -11,11 +11,15 @@
 import com.pgms.coredomain.domain.member.Member;
 
 public interface MemberRepository extends JpaRepository<Member, Long> {
+
 	boolean existsByRoleId(Long roleId);
 
 	@EntityGraph(attributePaths = {"role"})
 	Slice<Member> findSliceBy(Pageable pageable);
 
+	@Query("select m from Member m join fetch m.role r where m.email = :email")
+	Optional<Object> findByEmailWithRole(String email);
```

Comment:
```suggestion
	Optional<Member> findByEmailWithRole(String email);
```
Member로 타입 명시해도 될 것 같아요~